src/schemas/OrganizationSchema.js

```diff
@@ -29,9 +53,72 @@ module.exports = (conn) => {
       type: String,
       trim: true,
     },
-    address: {
+    // It refers to the economic activity carried out
+    industry: {
+      type: String,
+      enum: [
+        'agriculture',
+        'banking',
+        'construction',
+        'consulting',
+        'mass consumption',
+        'education',
+        'energy',
+        'pharmaceutical/laboratories',
+        'financial',
+        'real estate',
+        'manufacturing',
+        'media outlets',
+        'mining',
+        'oil industry',
+        'retail',
+        'health',
+        'public sector',
+        'public services',
+        'technology',
+        'telecommunications',
+        'transport',
+        'tourism',
+      ],
+    },
+    // Classification of your technological development
+    category: {
+      type: String,
+      enum: [
+        'digital transformation',
+        'software factory',
+        'agency',
+        'small and medium sized business',
+        'high tech',
+      ],
+    },
+    // Subclassification depending on the category
+    subcategory: {
+      type: String,
+      enum: [
+        'early stage corporation', // category: 'digital transformation'
+        'in-develop corporation', // category: 'digital transformation'
+        'mature corporation', // category: 'digital transformation'
+        'tech consulting firm', // category: 'software factory'
+        'software boutique', // category: 'software factory'
+        'ux/ui agency', // category: 'agency'
+        'digital agency', // category: 'agency'
+        'traditional business', // category: 'small and medium sized business'
+        'startup', // category: 'small and medium sized business'
+      ],
+    },
+    size: {
+      type: String,
+      enum: [
+        '50 - 500 employees',
+        '500 - 1,000 employees',
+        '1,000 - 10,000 employees',
+        '10,000+ employees',
```

Comment:
Algo como esto, los adjetivos son subjetivos y pueden cambiarlos según su nuestro contexto y criterio, pero pienso que usandolos ganas flexibilidad para identificarlos en tus archivos para `intl`

```suggestion
        'small',
        'medium',
        'big',
        'corporate',
```

Reply:
Listo, sugerencia tomada en cuenta.